src/Analytics/Adapter/Plausible.php

```diff
@@ -89,10 +90,34 @@ public function __construct(string $domain, string $apiKey, string $useragent, s
         $this->clientIP = $clientIP;
     }
 
+    /**
+     * Sets the useragent to use for requests.
+     * 
+     * @param string $userAgent The useragent to use for requests.
+     * 
+     * @return void
+     */
+    public function setUserAgent(string $userAgent): void
```

Comment:
Don't we need to implement these methods for orbit and active campaign as well ?

Reply:
Orbit and ActiveCampaign do not need user-agents since they tie all data to contacts

Reply:
what about the IP addresses ? Don't Orbit and Active Campaign respect the referrer header ? Can you check ?

Reply:
Neither of them respect the referrer header, activecampaign can receive the IP address if we create a custom field for the contact though